homework_6_191122/src/Triangle.java

```diff
@@ -25,25 +21,19 @@ public double getSideB() {
         return sideB;
     }
 
-    public double setBaseTriangle(double baseTriangle) {
-        return this.baseTriangle = baseTriangle;
+    public double setSideC(double sideC) {
+        return this.sideC = sideC;
     }
 
-    public double getBaseTriangle() {
-        return baseTriangle;
+    public double getSideC() {
+        return sideC;
     }
 
     public double getPerimeter() {
-        return this.perimeter = sideA + sideB + baseTriangle;
+        return sideA + sideB + sideC;
     }
 
-
-    public double getHeightTriangle() {
-        return this.heightTriangle = 2 / baseTriangle * Math.sqrt((perimeter / 2) * (perimeter / 2 - sideA) * (perimeter / 2 - sideB) * (perimeter / 2 - baseTriangle));
-    }
-
-
     public double getAreaTriangle() {
-        return this.areaTriangle = 1.0 / 2.0 * baseTriangle * heightTriangle;
+        return Math.sqrt(((sideA + sideB + sideC) / 2) * ((sideA + sideB + sideC) / 2 - sideA) * ((sideA + sideB + sideC) / 2 - sideB) * ((sideA + sideB + sideC) / 2 - sideC));
```

Comment:
(sideA + sideB + sideC) / 2) value is repeated 4 times, create new variable halfPerimeter and use it instead

Reply:
Fixed. Created new variable halfPerimeter and used it in getAreaTriangle() method.